AirCasting/Utils/Textfield.swift

```diff
@@ -7,27 +7,27 @@
 
 import SwiftUI
 
-func createTextfield(placeholder: String, binding: Binding<String>) -> some View {
+func createTextfield(placeholder: String, binding: Binding<String>, checkingIf: Bool = false) -> some View {
     TextField(placeholder,
               text: binding)
     .padding()
     .frame(height: 50)
     .overlay(
         RoundedRectangle(cornerRadius: 8)
-            .stroke(Color.aircastingGray.opacity(0.1), lineWidth: 1)
+            .stroke(checkingIf ? .red : Color.aircastingGray.opacity(0.1), lineWidth: 1)
     )
     .background(RoundedRectangle(cornerRadius: 8).fill(Color.aircastingGray.opacity(0.05)))
 }
 
-func createSecuredTextfield(placeholder: String, binding: Binding<String>) -> some View {
+func createSecuredTextfield(placeholder: String, binding: Binding<String>, checkingIf: Bool = false) -> some View {
```

Comment:
maybe `shouldWarnUser`? if not please omit

Reply:
I love this idea, thanks!